resources/sushy-tools/Dockerfile

```diff
@@ -1,7 +1,15 @@
 FROM registry.hub.docker.com/library/python:3.9
 
+ENV SUSHY_TOOLS_VERSION="0.18.1"
+
 RUN apt update && \
     apt install -y libvirt-dev && \
-    pip3 install sushy-tools==0.18.1 libvirt-python
+    apt clean && \
+    pip3 install --no-cache-dir sushy-tools==${SUSHY_TOOLS_VERSION} libvirt-python
+
+ENV SUSHY_EMULATOR_CONFIG="/root/sushy/conf.py"
```

Comment:
Is this really useful given that it's inside a container?

Reply:
The thing is if you want to specify a custom config you need to overwrite the CMD as:

```
sudo podman run -d --privileged --rm --name sushy-tools -v /home/core/sushy-tools/sushy-emulator.conf:/etc/sushy/sushy-emulator.conf:Z -v /var/run/libvirt:/var/run/libvirt:z -e SUSHY_EMULATOR_CONFIG=/etc/sushy/sushy-emulator.conf -p 8000:8000 quay.io/metal3-io/sushy-tools:latest sushy-emulator
```

otherwise it will use the default `/root/sushy/conf.py` file instead and will fail:

```
Traceback (most recent call last):
  File "/usr/local/bin/sushy-emulator", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/sushy_tools/emulator/main.py", line 773, in main
    app.configure(config_file=args.config)
  File "/usr/local/lib/python3.9/site-packages/sushy_tools/emulator/main.py", line 85, in configure
    self.config.from_pyfile(config_file)
  File "/usr/local/lib/python3.9/site-packages/flask/config.py", line 119, in from_pyfile
    with open(filename, mode="rb") as config_file:
FileNotFoundError: [Errno 2] Unable to load configuration file (No such file or directory): '/root/sushy/conf.py'
```

Reply:
Cannot you mount if over the existing path? Anyway, it's not an objection, I'm just asking.

Reply:
You are right, this works:

```
sudo podman run -d --privileged --rm --name sushy-tools -v /home/core/sushy-tools/sushy-emulator.conf:/root/sushy/conf.py:Z -v /var/run/libvirt:/var/run/libvirt:z -p 8000:8000 quay.io/metal3-io/sushy-tools:latest
```

Reply:
So I guess it would be better then to have something like:

```
ENV SUSHY_EMULATOR_LISTEN_IP="::"
ENV SUSHY_EMULATOR_LISTEN_PORT="8000"
ENV SUSHY_EMULATOR_CONFIG="/root/sushy/conf.py"
ENV SUSHY_EMULATOR_EXTRA_FLAGS="--debug"

ENTRYPOINT /usr/local/bin/sushy-emulator
CMD ${SUSHY_EXTRA_FLAGS}
```

Or should I get rid of all those flags and keep the CMD as it is now?

Thanks!